public/css/movie.css

```diff
@@ -50,30 +40,28 @@
 }
 
 .right-container iframe {
-  border-radius: 20px;
-  height: 480px;
-  /*box-shadow: 5px 5px 10px #97979793;*/
+  height: 435px;
 }
 
 .rating-container {
   display: flex;
   flex-direction: row;
   align-items: center;
-  margin-top: 20px;
 }
 
 .rating-container.individual-movie i {

```

Comment:
@AmeliaLim changing this selector to ".rating-container.individual-movie i:not(.filled)" will avoid that strange overlay (unless that's how you wanted it)